packages/extension-ui/src/Popup/Accounts/CreateAccountMenu.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ThemeProps } from '../../types';
+
+import React from 'react';
+import styled from 'styled-components';
```

Comment:
Could we always start imports order from external libraries? 🙂

Reply:
![image](https://user-images.githubusercontent.com/50427204/215439816-07ccf46d-d709-40e0-b6c1-5ffdc21ce9b1.png)